README.md

```diff
@@ -1,23 +1,25 @@
-# Morphos Server
+Morphos Server
+===============
 
-## Description
+![tests](https://github.com/danvergara/dblab/actions/workflows/test.yaml/badge.svg)
 
-Self-Hosted file converter server.
+__Self-Hosted file converter server.__
 
-## Goals of the project
+## Table of contents
 
-The goals is the creation of a Self-Hosted file converter server to be sovereing on the file manaement. 
-The most of the file converter websites claims tha they never keep our files longer than a short period of time, but we can't be so sure if they really keep that promise. Even if so, they can read and process the content of our files in that period of time. We are better off if our files never get the cloud (other people's computers), specially if our files hold very private or legal data. Morphos aims to solve that by providing a highly portable and configurable file converter server.
+- [Overview](#overview)
+- [Features](#features)
+- [License](#license)
 
-## Distribution
+## Overview
 
-The project is meant to be distributed by Golang binary embeding all the required assets.
-A Container image can be provided as well, but for the most of the cases the compiled binary will be enough, so the lack of container tecnologies installed on the host machine will not be an impediment to run the project.
+Today we are forced to rely on third party services to convert files to other formats. This is a serious threat to our privacy, if we use such services to convert files with highly sensitive personal data. It can be used against us, sooner or later.
+Morphos server aims to solve the problem mentioned above, by providing a self-hosted server to convert files privately. It provides a nice UI, so it's easy to use.
```

Comment:
I think  `friendly` or  `user-friendly` could match a better description for the UI

Reply:
What about this below?

"The project provides an user-friendly web UI."